ARSIS-Telemetry-GroundControl/groundcontrol-db/database.sql

```diff
@@ -31,4 +31,13 @@ CREATE TABLE IF NOT EXISTS biometrics (
   updatedAt TIMESTAMP DEFAULT now()
 );
 
+CREATE TABLE IF NOT EXISTS uia (
+  id SERIAL PRIMARY KEY,
```

Comment:
I think we need to add the uuid and then make it so we pull the most recent thing so we can have logging of when things changed.